iroh/src/commands/sync.rs

```diff
@@ -123,6 +123,21 @@ pub enum DocCommands {
         #[clap(short, long)]
         doc: Option<NamespaceId>,
     },
+    /// Leave a document, optionally removing it from this node.
+    Leave {
```

Comment:
I wonder if this is backwards/if we should have a regular `Remove` command, for when you just want to delete a doc, no matter the state. The `Leave` wording for me personally is pretty confusing if this is my own document

Reply:
This was suggested by @b5 in #1460